tests/tools/test_metadata.py

```diff
@@ -0,0 +1,62 @@
+import anndata
+import numpy as np
+import pandas as pd
+from scipy import sparse
+
+import pertpy as pt
+
+# Random generate data settings
+num_cells = 100
+num_genes = 100
+num_cells_per_id = num_cells // 4
+
+
+class TestMetaData:
+    def make_test_adata(self):
+        np.random.seed(1)
+        # generate count matrix
+        np.random.seed(1)
+        for i in range(num_genes):
```

Comment:
This looks a bit complicated to me. Isn't it possible to do this more simply by specifying a shape with random.normal or random.binomial or whatever?

What is the reason for the complexity of this setup?